src/modules/claims/claims.service.ts

```diff
@@ -599,15 +632,15 @@ export class ClaimsService {
       const [issuedToken, vp] = await Promise.all([
         this._didRegistry.issuePublicClaim({
           publicClaim,
-          expirationTimestamp,
+          expirationTimestamp: claimExpirationTimestamp,
```

Comment:
Are you sure it makes sense to pass `expirationTimestamp` even if `claimExpirationTimesamp === undefined`. To me that is relying a bit too much on the implementation of `issuePublicClaim` to not add an `exp` to the JWT if an `undefined` value is passed (which I think is the behaviour we would want `issuePublicClaim` to have)

Reply:
@Harasz Are you also sure in this case that the expiration wouldn't be added to the JWT if it is undefined?

Reply:
Yes, i am sure. For javascript both cases are the same. lets do simple test
```
const a = {prop1: undefined}
console.log(a.prop1) // undefined
console.log(a.prop2) // no, error just unefined
console.log(a) // { prop1: undefined }
```
The only difference is that third console log show prop1

Reply:
Ok, I was concerned that this could result in JWTs as shown below (as this would be an [invalid JWT](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4), I believe) 
```json
{
  "exp": undefined
}
```
But I realized now that this isn't even valid JSON, so think that this pretty unlikely to happen